kubeshop/monokle
# Olelensmar/feat/show patch links

This PR highlights linked patch files in the navigator and editor - linked resource/files are clickable which selects the corresponding file or resource


## Changes

-

## Fixes

#333 

## How to test it

Load the argo-rollouts manifests folder and select the namespace-install kustomization - both resource and file patches should be shown in refs-popup and editor

## screenshots

-

## Checklist
